UITests/Responses/TransferFunds/ListTransferMethodsNoData.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
You don't need No Data json file. It will have nothing not even count and other details